File: app/src/main/java/cn/intret/app/picgo/model/event/DeleteFolderMessage.java
package cn.intret.app.picgo.model.event;


import java.io.File;

/**
 * A folder has been delete.
 */
public class DeleteFolderMessage extends BaseMessage {
    File mDir;


    public DeleteFolderMessage(String desc, File dir) {
        super(desc);
        mDir = dir;
    }

    public DeleteFolderMessage(File dir) {
        mDir = dir;
    }

    public File getDir() {
        return mDir;
    }

    public DeleteFolderMessage setDir(File dir) {
        mDir = dir;
        return this;
    }

    @Override
    public String toString() {
        return "DeleteFolderMessage{" +
                "mDir=" + mDir +
                "} " + super.toString();
    }
}

File: app/src/main/java/cn/intret/app/picgo/model/event/DeleteFolderMessage.kt
package cn.intret.app.picgo.model.event


import java.io.File

/**
 * A folder has been delete.
 */
class DeleteFolderMessage : BaseMessage {
    internal var mDir: File


    constructor(desc: String, dir: File) : super(desc) {
        mDir = dir
    }

    constructor(dir: File) {
        mDir = dir
    }

    fun getDir(): File {
        return mDir
    }

    fun setDir(dir: File): DeleteFolderMessage {
        mDir = dir
        return this
    }

    override fun toString(): String {
        return "DeleteFolderMessage{" +
                "mDir=" + mDir +
                "} " + super.toString()
    }
}
